docs/markdown/IDE-integration.md

```diff
@@ -93,13 +93,21 @@ can provide code completion for all source files.
 ```json
 {
     "language": "language ID",
+    "machine": "build" / "host",
```

Comment:
At a quick look, the other parts of this JSON snippet are valid JSON, while the one added here is not. I don't know how relevant this is but it may be confusing to have invalid example data. Unfortunately JSON does not allow comments, thus I don't really know how to fix this without dropping the information that `machine` is an enumerated type.